flows/nodes/scriggo.md

```diff
@@ -0,0 +1,378 @@
+* No `unsafe` package and limited pointer operations. The `unsafe` package cannot be imported and pointer arithmetic will likely result in unexpected results.
+* No `defer` operation.  Sorry, not implemented, do it like it's C99!
+
+## Configuration
+
+* `Code`, required: The Go code to run.
+* `Outputs`: a list of variables to be output from the program (see below).
+
+## Output
+
+The Go node can modify any value in the incoming payload. It can also create new items in the outgoing payload, but those new items *must* be listed in the Outputs config field.
+
+Variables in the payload are accessed in Go code by using the Gettter/Setter interface on the `Payload` variable exported in the `gravwell` package, e.g. `gravwell.Payload.Set("foobar", 3.14159)`.
+
+### Debug Output
+
+All `stdout` or `stderr` output will be written to the Flow console log. This means including a `println("hello mom!")` will include `hello mom!` in the flow debug output.
+
+## Native Gravwell Package
+
+The Go flow node can import a package called `gravwell` that will allow access to flow specific functions and values.  The `gravwell` package can almost be thought of as a "virtual" package because it simply maps global variables and some internal flow functionality to a package so that Go node programs can interact with flow variables.
+
+The `gravwell` package exposes the following variables and functions:
+
+### Payload Getter and Setter
+
+One of the most useful variables in the `gravwell` package is the `Payload` variable which enables getting and setting natively typed values in the Flow payload.  The `Payload` variable implements the following interface:
+
+```
+type Payload interface {
+	Get(string) (interface{}, error)
+	Set(string, interface{}) error
+}
+```
+
+The `Payload` interface can get and set native types into the flow payload similar to the [Javascript](/flows/nodes/javascript) node or other logic nodes.
+
+```{note}
+Defining a new struct type in your Go program and setting it into the flow Payload may result in undefined behavior, because that type information won't make it outside the Go flow node.
+```
+
+### Other Package Types
+
+| Name     | Type/Prototype      | Description                                                    |
+|----------|---------------------|----------------------------------------------------------------|
+| `START`    | [time.Time](https://pkg.go.dev/time#Time) | The scheduled start time for the flow. This time may be far in the past if the flow is backfilling missed runs. |
+| `LAST_RUN` | [time.Time](https://pkg.go.dev/time#Time) | The timestamp of the last run of this flow. |
+| `DURATION` | [time.Duration](https://pkg.go.dev/time#Duration) | The scheduled time duration of a flow run. This repesents the time window the flow should cover for any queries. |
+| `ThisScriptID` | `string` | The current flow ID |
```

Comment:
All the other descriptions end with a period.

Reply:
good catch.